docs/tutorials/setting-up-an-nfs-file-share.md

```diff
@@ -97,56 +97,54 @@ After creating the file, the `nfsd` daemon [should automatically start up][OSXSe
 
 If you make changes to `/etc/exports`, activate them with `nfsd update`.
 
-Now update the fstab on the EV3.  
-
 ## How To Do It - EV3
 
-On the EV3 we first need to enable and start NFS modules. Type these commands on the command line:
-
-    sudo systemctl enable nfs-common.service
-    sudo systemctl start nfs-common.service
-    sudo systemctl enable rpcbind.service
-    sudo systemctl start rpcbind.service
-
-Next you'll need to update a file (as root) called `/etc/fstab`. You should have already set up USB Networking, so `ssh` to the EV3 and run an editor like `vi` or `nano` to edit the file. Here's the line you want to add to `/etc/fstab` - DO NOT TOUCH ANYTHING ELSE IN THERE!
-
+On the client - ev3dev - side, we need to create a system mount unit in order to mount our newly created NFS share. **Note**: The classic way of mounting the NFS share via an entry in /etc/fstab does not work!
 
-    # NOTE - the following examples all use the same IP address for the host, in practice, there would
-    #        be separate addresses for each host!
+First off, we need the NFS kernel module. Open an SSH connection to ev3dev and type the following:
 
-    # For the Linux example, it would look like:
-    192.168.2.1:/home/hostuserid/nfs/ev3dev /home/robot/nfs/linux   nfs users,noauto,rw,vers=3  0 0
+    modprobe nfs
```

Comment:
This needs to be run a root:

    sudo modprobe nfs

Reply:
Although I am wondering if this needs to be done at all. Did you try without loading the module first to see if it is automatically loaded?

Reply:
I didn't try that. Can't hurt to do it explicitely, can it?

Reply:
You're right, this is done automatically. I'll remove the entire section.